pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/slack/index.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: layout.pug
+navigationTitle: Sending Alerts via Slack
+title: Sending Alerts via Slack
+menuWeight: 20
+excerpt: Configuring Alertmanager to send Slack notifications
+render: mustache
+model: ../../../data.yml
+---
+
+# Configuring Alertmanager to alert via Slack
+
+## Create an Alertmanager configuration file
+
+See the [Alertmanager configuration documentation](https://prometheus.io/docs/alerting/configuration/) and the [Slack configuration section](https://prometheus.io/docs/alerting/configuration/#slack_config) for a detailed overview of how to create a configuration file with a Slack receiver.
+
+See the following example of a basic configuration file that's set up with a Slack receiver.
```

Comment:
"that's" should be avoided.

Suggest we reword this as:

"Below is an example of a simple configuration file with a Slack receiver"